codec/types/any_test.go

```diff
@@ -28,6 +28,9 @@ var eom = &errOnMarshal{}
 // Ensure that returning an error doesn't suddenly allocate and waste bytes.
 // See https://github.com/cosmos/cosmos-sdk/issues/8537
 func TestNewAnyWithCustomTypeURLWithErrorNoAllocation(t *testing.T) {
+	// make sure we're not in the middle of a GC.
+	runtime.GC()
```

Comment:
So the fix is just to run garbage collection prior to future runtime calls, I see.